src/filetest/binemit.rs

```diff
@@ -72,12 +72,18 @@ impl binemit::CodeSink for TextSink {
         write!(self.text, "{}({}) ", reloc, ebb_offset).unwrap();
     }
 
-    fn reloc_external(&mut self, reloc: binemit::Reloc, name: &ir::ExternalName) {
+    fn reloc_external(
+        &mut self,
+        reloc: binemit::Reloc,
+        name: &ir::ExternalName,
+        addend: binemit::Addend,
+    ) {
         write!(
             self.text,
-            "{}({}) ",
+            "{}({}{:+}) ",
```

Comment:
Could we print the `Addend`  the same way we print an `Offset32`? That is, print nothing rather than `+0` for a null addend.

Reply:
Sure. I updated the patch.